contracts/Party.sol

```diff
@@ -158,17 +158,12 @@ contract Party is ReentrancyGuardUpgradeable, ERC721HolderUpgradeable {
 
     function __Party_init(
         address _nftContract,
-        uint256 _tokenId,
         Structs.AddressAndAmount calldata _split,
         Structs.AddressAndAmount calldata _tokenGate,
         string memory _name,
         string memory _symbol
     ) internal {
         require(msg.sender == partyFactory, "Party::__Party_init: only factory can init");
-        // validate token exists (must set nftContract & tokenId before _getOwner)
-        nftContract = IERC721Metadata(_nftContract);
```

Comment:
in this case -- maybe we could call a fn that validates that nftContract does in fact adhere to the 721 standard?

Reply:
Hmmm. okay, thinking out loud about this.. the following functions are shared with ERC-20:
- `transferFrom`
- `balanceOf`
- `approve`


So the options unique to ERC-721 would be the following:
- `ownerOf`
- `getApproved`
- `isApprovedForAll`
- `setApprovalForAll` (write) 
- `safeTransferFrom` (write)

of the top three read functions, we would have to supply some tokenId input to try to call the function.. I guess we could try to always pass in `ownerOf(tokenId = 1)` but if that `tokenId` is not assigned for some reason, most NFT contracts would throw. same for `getApproved(tokenId)` -- throws if `tokenId` is not a valid NFT

i guess we could do `isApprovedForAll(partyFactory, partyDAOMultisig)`